website/docs/docs/core/connect-data-platform/risingwave-setup.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "RisingWave setup"
+id: "risingwave-setup"
+description: "Read this guide to learn about how to set up RisingWave in dbt."
+meta:
+  maintained_by: RisingWave
+  pypi_package: 'dbt-risingwave'
+  authors: 'Dylan Chen'
+  github_repo: 'risingwavelabs/dbt-risingwave'
+  min_core_version: 'v1.6.1'
+  min_supported_version: ''
+  cloud_support: Not Supported
+  slack_channel_name: '#all-things-risingwave'
+  slack_channel_link: 'https://www.risingwave.com/slack'
+  platform_name: 'RisingWave'
+  config_page: '/reference/resource-configs/no-configs'
+---
+
+:::info Vendor-supported plugin
+
+Certain core functionality may vary. If you would like to report a bug, request a feature, or contribute, you can check out the linked repository and open an issue.
+
+:::
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta} />
+
+## Connecting to RisingWave with dbt-risingwave
+
+Before connecting to RisingWave, ensure that RisingWave is installed and running. For more information about how to get RisingWave up and running, see the [RisingWave quick start guide](https://docs.risingwave.com/docs/dev/get-started/).
+
+To connect to RisingWave with dbt, you need to add a RisingWave profile to your dbt profile file (`~/.dbt/profiles.yml`). Below is an example RisingWave profile. Revise the field values when necessary.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+default:
+  outputs:
+    dev:
+      type: risingwave
+      host: 127.0.0.1 
```

Comment:
```suggestion
      host: <name_number>
```

Is the host always the same?

Reply:
I think we can follow the dbt documentation. https://docs.getdbt.com/docs/core/connect-data-platform/postgres-setup
```
host: [hostname]
user: [username]
password: [password]
port: [port]
dbname: [database name] 
schema: [dbt schema]
```